plejd/PlejdApi.js

```diff
@@ -436,6 +436,8 @@ class PlejdApi {
             loadType = 'light';
           }
 
+          const room = this.siteDetails.rooms.find((x) => x.roomId === device.roomId);
```

Comment:
`room.x` (below) will crash if room is not found. Recommend appending a default value
`= this.siteDetails.....  || {}` 

Alt is to always set similar to `room: room ? room.title : undefined`

Reply:
Point taken, but competency is lacking on my side.
Trial, error and googling did not help.
Can you provide some more details how to exactly do this. Sorry for asking.

The rest of the comments I should be proficient to handle :-)

Reply:
Got it, finally.